renovate.json

```
@@ -1,46 +1,87 @@
-   }
- ],
- "prHourlyLimit": 5,
- "prConcurrentLimit": 5,
- "rangeStrategy": "bump",
- "renovateFork": true,
- "includeForks": true,
- "regexManagers": [
+  "extends":[
+    "config:base"
+  ],
+  "enabled": "true",
+  "baseBranches": ["master"],
+  "timezone": "Europe/Helsinki",
+  "packageRules": [
     {
-      "fileMatch": ["^lib/common.sh$"],
-      "matchStrings": ["MINIKUBE_VERSION:-\"(?<currentValue>.*?)\"}"],
-      "depNameTemplate": "kubernetes/minikube",
-      "datasourceTemplate": "github-releases"
+      "depTypeList": ["devDependencies"],
+      "matchUpdateTypes": ["minor"],
+      "automerge": true
     },
     {
-      "fileMatch": ["^lib/common.sh$"],
-      "matchStrings": ["KIND_VERSION:-\"(?<currentValue>.*?)\"}"],
-      "depNameTemplate": "kubernetes-sigs/kind",
-      "datasourceTemplate": "github-releases"
+      "datasources": ["docker"],
+      "packageNames": ["kindest/node"],
+      "automerge": true 
+    },    
+    {
+      "datasources": ["pypi"],
+      "packageNames": ["ansible"],
+      "automerge": true 
     }
+
   ],
-  "assignees": ["fmuyassarov", "kashifest"],
-  "assigneesSampleSize": 1,
-  "reviewers": [
-      "furkatgofurov7",
-      "jan-est",
-      "lentzi90",
-      "macaptain",
-      "mboukhalfa",
-      "namnx228",
-      "smoshiur1237",
-      "Sunnatillo",
-      "wgslr",
-      "Xenwar"
-  ],
-  "reviewersSampleSize": 2
-}
+  "prHourlyLimit": 5,
+  "prConcurrentLimit": 5,
+  "rangeStrategy": "bump",
+  "renovateFork": true,
+  "includeForks": false,
+  "stabilityDays":"5",
+  "regexManagers": [
+     {
+       "fileMatch": ["^lib/common.sh$"],
+       "matchStrings": ["KUBERNETES_VERSION:-\"(?<currentValue>.*?)\"}"],
+       "depNameTemplate": "kubernetes/kubernetes",
+       "datasourceTemplate": "github-releases"
+     },
+     {
+       "fileMatch": ["^lib/common.sh$"],
+       "matchStrings": ["MINIKUBE_VERSION:-\"(?<currentValue>.*?)\"}"],
+       "depNameTemplate": "kubernetes/minikube",
+       "datasourceTemplate": "github-releases"
+     },
+     {
+       "fileMatch": ["^lib/common.sh$"],
+       "matchStrings": ["KIND_VERSION:-\"(?<currentValue>.*?)\"}"],
+       "depNameTemplate": "kubernetes-sigs/kind",
+       "datasourceTemplate": "github-releases"
+     },
+     {
+       "fileMatch": ["^lib/common.sh$"],
+       "matchStrings": ["KUSTOMIZE_VERSION:-\"(?<currentValue>.*?)\"}"],
+       "depNameTemplate": "kubernetes-sigs/kustomize",
+       "datasourceTemplate": "github-releases"
+     },
+     {
+       "fileMatch": ["^lib/common.sh$"],
+       "matchStrings": ["ANSIBLE_VERSION:-\"(?<currentValue>.*?)\"}"],
+       "depNameTemplate": "ansible",
+       "datasourceTemplate": "pypi"
+     },
+     {
+      "fileMatch": ["^lib/common.sh$"],
+      "matchStrings": ["KIND_NODE_IMAGE_VERSION:-\"(?<currentValue>.*?)\"}"],
+      "depNameTemplate": "kindest/node",
+      "datasourceTemplate": "docker"
+     }      
+   ],
+   "assignees": ["fmuyassarov", "kashifest"],
```

since @furkatgofurov7 moved recently to the approvers list
```suggestion
   "assignees": ["fmuyassarov", "furkatgofurov7", "kashifest"],
```